app/src/main/java/zapsolutions/zap/fragments/WalletFragment.java

```diff
@@ -266,6 +266,10 @@ public void onSingleClick(View v) {
             connectionToLNDEstablished();
         } else {
             if (PrefsUtil.isWalletSetup()) {
+                if (!LndConnection.getInstance().isConnected()) {
```

Comment:
It's possible that we get to here before the `onMoveToForeground` has been called, which means we do not have a connection opened yet, so if that is the case, we do it here, so we can safely call `checkIfLndIsReachableAndTriggerWalletLoadedInterface`